### DISCUSSION TITLE
BTF-LIGHTING Zigbee 3.0 C02Z LED

### DISCUSSION THREAD

**Discussion by leoogermenia** [Koenkk/zigbee2mqtt#21068] [Devices]
Hi everyone,
is anyone know if this [product](https://amzn.eu/d/49CfWEI) is compatible with zigbee2mqtt?
If this cannot be integrated, there is any alternatives? The [lightstrip is a dimmable CCT](https://www.btf-lighting.com/collections/cct-fcob-5v-12v-24v/products/fcob-cct-led-light-strip-640-leds-high-density-flexible-fob-cob-10mm-led-lights-ra90-warm-white-with-white-linear-dimmable-dc24v?variant=44476550840546)

**Comment by xavivars** [comment]
I have successfully integrated it with zigbee2mqtt, and you can change brightness as well as color temp from zigbee2mqtt's UI. However, I have no idea how to expose that functionality to home assistant: I can only switch it on and off, and trigger effects.

![image](https://github.com/Koenkk/zigbee2mqtt/assets/134083/4c100147-a694-4dd1-a90c-87ac6706fa3a)

**Reply by leoogermenia** [reply]
Oh perfect!
Can you share the state of the entity on Home Assistant? Go on developer tools -> states and search the entity with the entity ID

**Reply by xavivars** [reply]
Figured it out: everything is exposed in Home Assisant, but directly (via entities) you only see "on/off". Then, when using the[ light card](https://www.home-assistant.io/dashboards/light/), you see brightness, but still not color temp. You need to click the three dots next to the card to then see the slider for temperature

**Reply by 877dev** [reply]
Hi all,

Has the C02Z been officially supported now?

**Reply by leoogermenia** [reply]
<img width="1159" alt="Screenshot 2024-10-29 alle 09 18 26" src="https://github.com/user-attachments/assets/b8897afb-2763-429f-af1a-527b88d35454">
Yes!

**Reply by hheimbuerger** [reply]
It auto-paired for me, and the full functionality seems available.

It was detected as a [MiBoxer FUT035Z+](https://www.zigbee2mqtt.io/devices/FUT035Z_.html) here.

(Different from OP, my device has no BTF branding on it. It's quite literally "white-label": it's almost all white, no manufacturer, but the model no. is stated as C02Z, which is how I got to this discussion.)
